SoftLayer/managers/ordering.py

```diff
@@ -370,6 +373,21 @@ def get_price_id_list(self, package_keyname, item_keynames):
 
         return prices
 
+    @staticmethod
+    def get_item_price_id(core, price, price_id):
+        """get item price id"""
+        category_code = []
```

Comment:
To make this a bit easier to read, do something like this

```python
    def get_item_price_id(core, price, price_id):
        """get item price id"""
        category_code = []
        capacity_min = int(price.get('capacityRestrictionMinimum', -1))
        capacity_max = int(price.get('capacityRestrictionMaximum', -1))
        if capacity_min is None:
            if price['categories'][0]['categoryCode'] not in category_code:
                category_code.append(price['categories'][0]['categoryCode'])
                price_id = price['id']
        elif capacity_min  <= int(core) <= capacity_max :
            if price['categories'][0]['categoryCode'] not in category_code:
                category_code.append(price['categories'][0]['categoryCode'])
                price_id = price['id']
        return price_id
```

Reply:
Done